app/helpers/sessions_helper.rb

```diff
@@ -35,6 +35,10 @@ def signed_in?
     !current_user.nil?
   end
 
+  def active_user?
+    current_user.active? unless current_user.nil?
```

Comment:
Maybe slightly easier to read with the "safe navigation" operator? `current_user&.active?